<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveFmtDialog</class>
 <widget class="QDialog" name="SaveFmtDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="fmtLabel">
       <property name="text">
        <string>Select for&amp;mat:</string>
       </property>
       <property name="buddy">
        <cstring>fmtSel</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fmtSel"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="ParamWidget" name="paramWidget" native="true"/>
     </item>
     <item>
      <widget class="PresetWidget" name="presetWidget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParamWidget</class>
   <extends>QWidget</extends>
   <header>mainwidgets/paramwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PresetWidget</class>
   <extends>QWidget</extends>
   <header>mainwidgets/presetwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SaveFmtDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SaveFmtDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fmtSel</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SaveFmtDialog</receiver>
   <slot>selFmt(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selFmt(int)</slot>
 </slots>
</ui>
